hivemind/hivemind_cli/run_server.py

```diff
@@ -108,8 +108,6 @@ def main():
         server.join()
     except KeyboardInterrupt:
         logger.info("Caught KeyboardInterrupt, shutting down")
-    finally:
```

Comment:
sanity check: is it true that this  line is merely unnecessary - and does not break anything?

Reply:
It is unnecessary, but the double server.shutdown works fine (we're somewhat lucky that it is though)